python/doc/examples/reliability_sensitivity/reliability/plot_form_explained.py

```diff
@@ -41,10 +42,11 @@
 graph_PDF.setLegendPosition("lower right")
 contours = graph_PDF.getDrawable(0).getImplementation()
 contours.setColorMapNorm("log")
```

Comment:
I noticed this example had problems too, I just removed the setColorMapNorm(log) call and everything was fine

Reply:
using rank instead solves the issue!